src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -25,170 +25,185 @@ const propTypes = {
-                            href="https://onfido.com/facial-scan-policy-and-release/"
-                            style={[styles.textMicro]}
-                        >
-                            {this.props.translate('onfidoStep.facialScan')}
-                        </TextLink>
-                        {', '}
-                        <TextLink
-                            href="https://onfido.com/privacy/"
-                            style={[styles.textMicro]}
-                        >
-                            {this.props.translate('common.privacy')}
-                        </TextLink>
-                        {` ${this.props.translate('common.and')} `}
-                        <TextLink
-                            href="https://onfido.com/terms-of-service/"
-                            style={[styles.textMicro]}
-                        >
-                            {this.props.translate('common.termsOfService')}
-                        </TextLink>
-                    </Text>
-                </Form>
-            </ScreenWrapper>
+            <RequestorOnfidoStep
+                reimbursementAccount={reimbursementAccount}
+                reimbursementAccountDraft={reimbursementAccountDraft}
+                onBackButtonPress={onBackButtonPress}
+                getDefaultStateForField={getDefaultStateForField}
+            />
         );
-    }
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <HeaderWithBackButton
+                title={translate('requestorStep.headerTitle')}
+                stepCounter={STEP_COUNTER}
+                guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
+                onBackButtonPress={onBackButtonPress}
+                shouldShowGetAssistanceButton
+            />
+            <Form
+                formID={ONYXKEYS.REIMBURSEMENT_ACCOUNT}
+                submitButtonText={translate('common.saveAndContinue')}
+                validate={validate}
+                onSubmit={submit}
+                style={STYLES.Form}
+                scrollContextEnabled
+            >
+                <Text>{translate('requestorStep.subtitle')}</Text>
+                <View style={STYLES.LearnMoreContainer}>
+                    <TextLink
+                        style={STYLES.LearnMoreLink}
+                        // eslint-disable-next-line max-len
+                        href="https://community.expensify.com/discussion/6983/faq-why-do-i-need-to-provide-personal-documentation-when-setting-up-updating-my-bank-account"
+                    >
+                        {`${translate('requestorStep.learnMore')}`}
+                    </TextLink>
+                    <Text style={STYLES.LearnMoreSeparator}>{' | '}</Text>
+                    <TextLink
+                        style={STYLES.DataSafeLink}
+                        href="https://community.expensify.com/discussion/5677/deep-dive-security-how-expensify-protects-your-information"
```

Comment:
Can we add it to `CONST.ts`? Moreover, there are other links there.

Reply:
That's a great idea! Is there any particular place inside CONST.ts that we should place the links?

Reply:
@kacper-mikolajczak, I see a whole block with URLs, you can put your right after those.
<img width="852" alt="Screenshot 2023-09-15 at 11 08 51" src="https://github.com/Expensify/App/assets/57314631/e9d08305-8c47-47d7-9ce4-17de45e9f474">